assets/dfn-panel.css

```diff
@@ -0,0 +1,60 @@
+/* dfn popup panel that list all local references to a dfn */
+dfn {
+  cursor: pointer;
+}
+.dfn-panel {
+  position: absolute;
+  left: var(--left); /* set via JS */
+  top: var(--top); /* set via JS */
+  z-index: 35;
+  height: auto;
+  width: max-content;
+  max-width: 300px;
+  max-height: 500px;
+  overflow: auto;
+  padding: 0.5em 0.75em;
+  font: small Helvetica Neue, sans-serif, Droid Sans Fallback;
+  background: #dddddd;
+  color: black;
+  border: outset 0.2em;
+}
+.dfn-panel * {
+  margin: 0;
+  padding: 0;
+  text-indent: 0;
+}
+.dfn-panel > b {
+  display: block;
+}
+.dfn-panel a[href] {
+  color: black;
+}
+.dfn-panel a[href].self-link {
+  display: inline-block;
+}
+.dfn-panel a:not(:hover) {
+  text-decoration: none !important;
+  border-bottom: none !important;
+}
+.dfn-panel a[href]:hover {
+  border-bottom-width: 1px;
+}
+.dfn-panel > b + b {
+  margin-top: 0.25em;
+}
+.dfn-panel ul {
+  padding: 0;
+}
+.dfn-panel li {
+  list-style: inside;
+}
+.dfn-panel.minimized {
+  display: inline-block;
+  position: fixed;
+  left: 0.5em;
+  top: unset;
+  bottom: 2em;
+  margin: 0 auto;
+  max-width: calc(100vw - 1.5em - 0.4em - 0.5em);
```

Comment:
Why this calculation? Some comment will be useful for future maintenance.

Reply:
I mostly copy-pasted style from bikeshed :sweat_smile:

Reply:
Hmm, maybe just replace it with precalculated value if we don't know it? 🤔

Reply:
Understood. 
```
0.75em from padding, 0.5em from left position
0.75em * 2 = 1.5em
0.5em * 2 = 1em
(total 2.5em - it was 2.4 for some reason)
```

Reply:
Not sure why we should do left * 2, maybe the 0.4em was from border * 2.

Reply:
Oh. good point!

Reply:
Corrected.